src/ContractEditor/index.js

```diff
@@ -12,43 +12,41 @@
  * limitations under the License.
  */
 
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
-import { MarkdownEditor } from '@accordproject/markdown-editor';
+import { SlateAsInputEditor } from '@accordproject/markdown-editor';
 import List from '@accordproject/markdown-editor/dist/plugins/list';
 import ClausePlugin from '../plugins/ClausePlugin';
-import ClauseComponent from '../components/ClauseComponent';
-
-const defaultMarkdown = `# Supply Agreement
-  This is a supply agreement between Party A and Party B.
-  
-  # Payment
-  
-  <clause src="https://templates.accordproject.org/archives/full-payment-upon-signature@0.7.1.cta" clauseid="skjh2342dsa">
-  Upon the signing of this Agreement, "Dan" shall pay the total purchase price to "Steve" in the amount of 0.01 USD.
-  </clause>
-  
-  ## Late Delivery And Penalty
-  
-  <clause src="https://templates.accordproject.org/archives/latedeliveryandpenalty@0.13.1.cta" clauseid="kgek32h4k3j2hkew">
-  Late Delivery and Penalty. In case of delayed delivery except for Force Majeure cases, "Dan" (the Seller) shall pay to "Steve" (the Buyer) for every 2 days of delay penalty amounting to 10.5% of the total value of the Equipment whose delivery has been delayed. Any fractional part of a days is to be considered a full days. The total amount of penalty shall not however, exceed 55% of the total value of the Equipment involved in late delivery. If the delay is more than 15 days, the Buyer is entitled to terminate this Contract.
-  </clause>
-  
-  End.
-  `;
 
 /**
  * Adds the current markdown to local storage
  */
 function storeLocal(value, markdown) {
+  console.log('MARKDOWN', markdown);
```

Comment:
I think this could be removed. Same with `src/plugins/ClausePlugin.js` `L223` and `L239`.